src/zambeze/campaign/transfer_activity.py

```diff
@@ -7,20 +6,15 @@
 from zambeze.orchestration.zambeze_types import MessageType, ActivityType
 
 
-class TransferActivity(Activity):
-    def __init__(self, name, source_file, dest_directory, override_existing=False):
+class TransferActivity:
+    def __init__(self, name: str, source_file, dest_directory, override_existing=False):
```

Comment:
why type hint just one arg?

Reply:
Oops, forgot to finish the type hints. I forgot the docstrings too. 🫤

Reply:
Done.